src/common/number/numericString.ts

```diff
@@ -1,79 +1,154 @@
+/**
+ *      aaaa0x111bbbbbb
+ *      |-------------| => NumericString
+ *      |--|            => prefix
+ *          |---|       => core
+ *               |----| => suffix
+ *          ||          => numPreffix
+ *            |-|       => num
+ *
+ * Parsing eagerly, leftmost match wins.
+ * If more than one match begins at the same position, the match with biggest
+ * span wins.
+ * If more than one match have the same begin and span (This usually happens
```

Comment:
grammar: more than one match *has* the same

Reply:
Fixed